index.html

```diff
@@ -163,11 +171,13 @@ <h2>Topic</h2>
               <img src="media/Js.png" alt="javaScript" />
             </li>
           </ul>
+          <button type="button" class="big-btn">See Project</button>
         </div>
-        <button type="button" class="big-btn">See Project</button>
+      
       </div>
     </div>
-    <div class="intro">
+    <div class="About">
```

Comment:
- [ ]  In **Desktop version**, The `About Myself` section should have the same design as the original website. Kindly fix the top-right design. Hint You can do that by using border-radius for design.
- Your website  
![gergsas](https://user-images.githubusercontent.com/26767383/154678015-1d32fc44-28c2-400f-b294-c6fbe020d1b2.PNG)

- Original website
![ngfbdf](https://user-images.githubusercontent.com/26767383/154678086-7b7002e1-de42-422e-bb5c-377446927031.PNG)